MetaMask/metamask-extension
# feat: improve blockaid error/warning copy

## **Description**
We should update content for:

when ppom returns warning as response

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24037?quickstart=1)

## **Related issues**

Fixes: [#1758](https://github.com/MetaMask/MetaMask-planning/issues/1758)

## **Manual testing steps**



1. Add this custom network: https://base.drpc.org with chainId 8453
2. Trigger a transaction from the test dapp -- see error
3. For warning, you can edit the code to return a warning alert response and see the warning.

## **Screenshots/Recordings**

### **Before**

<img width="353" alt="Screenshot 2024-04-15 at 22 37 08" src="https://github.com/MetaMask/metamask-extension/assets/44811/2572b1a5-233d-4e7e-83c3-5ea92faef362">


### **After**

<img width="337" alt="Screenshot 2024-04-15 at 22 36 21" src="https://github.com/MetaMask/metamask-extension/assets/44811/1e4ec760-3714-4a91-9e5b-e39784887354">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
